packages/core/src/utilities/color/_color.docs.scss

```diff
@@ -3,10 +3,10 @@ Color
 
 Use the color utility to change an element's text color.
 
-**Format**: `ds-u-color--[COLOR NAME]-[SHADE]`
-
 The options below offer color palette combinations that fall within the range of Section 508 compliant foreground/background color contrast ratios. To ensure that text remains accessible, **use only these permitted color combinations**.
 
+**Format**: `ds-u-color--[COLOR NAME]-[SHADE]`
+
 @hide-markup
```

Comment:
yeah i didn’t change this one much, i’m not sure how handle it. we have specific guidance that we need to use those color combos, so that’s why i don’t know if we can remove combos. i was considering grouping by background color, but then later on there are combos that are grouped by text color...